modules/freewheelSSPBidAdapter.js

```diff
@@ -223,6 +223,24 @@ export const spec = {
       componentId: getComponentId(currentBidRequest.params.format)
     };
 
+    // Add GDPR flag and consent string
+    if (bidderRequest.gdprConsent) {
+      requestParams._fw_gdpr_consent = bidderRequest.gdprConsent.consentString;
+
+      if (typeof currentBidRequest.gdprConsent.gdprApplies === 'boolean') {
+        requestParams._fw_gdpr = currentBidRequest.gdprConsent.gdprApplies;
+      }
+    }
+
+    var vastParams = currentBidRequest.params.vastUrlParams;
+    if (typeof vastParams === 'object') {
+      for (kye in vastParams) {
```

Comment:
Seems to be a small typo here; believe it should be `key`

Reply:
done, thx